net/iperf3/Portfile

```diff
@@ -35,19 +35,13 @@ if {${subport} eq ${name}} {
                         sha256  9b03a51623d2098eddbd04f8802d1d03def5571a0fcd97b3ce5b51b52b07d84d \
                         size    652099
 
-    conflicts           ${name}-devel
-
     github.livecheck.regex {([0-9.]+)}
 }
 
 subport ${name}-devel {
-    github.setup        esnet iperf 40a663b2871dce1da4cee571a62e43d3557e6816
-    version             20230719
+    PortGroup           obsolete 1.0
+    version             20231123
     revision            0
 
-    checksums           rmd160  97b1a842b052557063d1456c84b8e9e189acaf30 \
-                        sha256  bcd79dcbe3dd4fc0987aa34e5ee99a80b0025cd012ab268491e4bfd842c871b5 \
-                        size    650883
-
-    conflicts           ${name}
+    replaced_by         ${name}
```

Comment:
please add comment along the lines of: 
# remove after November 24, 2024

that signifies the 1-year period when the subport can be completely removed.

Reply:
I added the comment.